### DISCUSSION TITLE
Low/Hight after audit fix

### DISCUSSION THREAD

**Discussion by ghost** [npm/rfcs#266] [RRFC] [answered]
When you get vulnerabilities warning, you know how many are 'low' or 'high' risk.

```
found 611 vulnerabilities (608 low, 3 high)
  run `npm audit fix` to fix them, or `npm audit` for details
```

Then you run `npm audit fix` to fix them. so if there some vulnerabilities need manual review you get that warning.

```
fixed 606 of 611 vulnerabilities in 1618 scanned packages
  5 vulnerabilities required manual review and could not be updated
```

I propose to add (_n_ low, _n_ high) so the user will need if there a vulnerabilities that are high and need immediate manual fix.

Thank you,

**Comment by ruyadorno** [comment, marked as answer]
hi @dragahassan thanks for the idea 😊 we discussed it briefly in our [OpenRFC meeting](https://github.com/npm/rfcs/blob/latest/meetings/2020-10-28.md#discussion-266-add-vuln-level-info-to-audit-fix-output---dragahassan) and the consensus is that this is small enough of an item that it doesn't need a full RFC.

It will become an item in our backlog or it can also be implemented by someone in the community.
